src/components/VIcon/VIcon.js

```diff
@@ -39,7 +39,7 @@ export default {
     const { small, medium, large, xLarge } = props
     const sizes = { small, medium, large, xLarge }
     const explicitSize = Object.keys(sizes).find(key => sizes[key] && key)
-    const fontSize = (explicitSize && SIZE_MAP[explicitSize]) || props.size
+    const fontSize = (explicitSize && SIZE_MAP[explicitSize]) || (props.size !== undefined && (props.size.indexOf('px') >= 0 ? props.size : props.size + 'px'))
```

Comment:
I think the `||` condition can be break into a new line

Reply:
We do this in other areas of the framework. Starting to think we need an abstracted helper function to run this through.

Reply:
This seems like a pretty messy way to do it too, VAvatar just uses `parseInt`: https://github.com/vuetifyjs/vuetify/blob/770a4962a155f21795de181a58923cd3df554dd0/src/components/VAvatar/VAvatar.js#L27

Reply:
@ebisbe I was talking about break it like [this line](https://github.com/vuetifyjs/vuetify/pull/3566/files#diff-655e9e29d68ab72a236746ac9af265b7R50) so the line length can be shorter.

Reply:
Sorry @peluprvi I change the name with @KaelWD, my fault. Should I brake the line for style, also?